docs/user/alerting/rule-management.asciidoc

```diff
@@ -62,6 +62,9 @@ image:images/bulk-mute-disable.png[The Manage rules button lets you mute/unmute,
 === Importing and exporting rules
 
 To import and export rules, use the <<managing-saved-objects, Saved Objects Management UI>>.
+After the succesful import the proper banner will be displayed:
+[role="screenshot"]
+image::images/rules-imported-banner.png[Rules import banner]
```

Comment:
This image is really big.  What would it look like if you changed the width:

```suggestion
image::images/rules-imported-banner.png[Rules import banner, width=50%]
```
Some of the other images in the file might benefit from a smaller width too.